config/sc-config.yaml

```diff
@@ -90,8 +93,12 @@ objectStorage:
       # - source: source-bucket
       #   # destination: destination-bucket # if unset it will be the same as 'source'
       #   # schedule: 0 5 * * *             # if unset it will be the same as 'defaultSchedule'
-      #   # sourceType: Optional but if included it uses s3 or swift
-      #   # destinationType: Optional but if included it uses s3 or swift.
+      #   # sourceType: #Optional but if included it uses s3 or swift
```

Comment:
Is this correct?
```suggestion
      #   # sourceType: # Optional but if included supported value are 's3' or 'swift'
```

Reply:
Reworded a lot of these comments now. Let me know if you want more updates